usr/sbin/p11sak/p11sak.c

```diff
@@ -460,8 +464,24 @@ static void print_gen_ec_help(void)
     printf("\n Usage: p11sak generate-key ec [ARGS] [OPTIONS]\n");
     printf("\n Args:\n");
     printf("      prime256v1\n");
-    printf("      secp384r1\n");
+    printf("      prime192\n");
+    printf("      secp224\n");
```

Comment:
I think secp384r1 got accidentally deleted

Reply:
And I saw that the naming of the prime192 curve is not consistent here: what we mean is NID_X9_62_prime192v1, but we usually use the short form secp192v1 or prime192v1. So I would change "prime192" to at least "prime192v1".

Reply:
Done